pkg/plugins/models.go

```diff
@@ -9,6 +9,8 @@ import (
 
 const (
 	TypeDashboard = "dashboard"
+
+	ActionAppAccess = "plugins.app:access"
```

Comment:
Apologies - way out of the loop on this but is the `"app"` in reference to the plugin type IE app plugins? I just see that technically for the CloudWatch datasource now for example, this will be returned as part of the API. I assume this is expected? I could be the only one lacking insight, but is this anything the plugin maintainers should be aware of to avoid any potential confusion?

Reply:
> is the "app" in reference to the plugin type IE app plugins?

Yes it is.

>  I just see that technically for the CloudWatch datasource now for example, this will be returned as part of the API. I assume this is expected?

No it's not :sweat_smile: 

Can data source plugins define `Includes`?